pages/coq-team.html

```diff
@@ -0,0 +1,503 @@
+    <div class="position">Inria Advanced Researcher in Saclay, France</div>
+    <div class="components">Mathematical Components, Notations</div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/bgregoir.jpg" alt="Benjamin Grégoire" width="170px"/></div>
+    <div class="name"><a href="http://www-sop.inria.fr/members/Benjamin.Gregoire/">Benjamin Grégoire</a></div>
+    <div class="nickname">bgregoir</div>
+    <div class="position">Inria Researcher in the Stamp Team, Sophia-Antipolis, France</div>
+    <div class="components">Ring, <verb>vm_compute</verb></div>
+  </div>
+  <div class="person">
+    <div class="picture"><img src="/files/team/vbgl.jpg" alt="Vincent Laporte" width="160px"/></div>
+    <div class="name"><a href="https://members.loria.fr/VLaporte/">Vincent Laporte</a></div><div class="nickname">vbgl</div>
+    <div class="position">Inria Researcher in the Pesto Team, Nancy, France</div>
+    <div class="role">RM: 8.10</div>
+    <div class="components">Nix, micromega</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/assia_mahboubi.jpg" alt="Assia Mahboubi" width="200px"/></div>
+    <div class="name"><a href="http://people.rennes.inria.fr/Assia.Mahboubi/">Assia Mahboubi</a></div><div class="nickname">amahboubi</div>
+    <div class="position">Inria Researcher in the Gallinette Team, Nantes, France</div>
+    <div class="role"><a href="https://cordis.europa.eu/project/id/101001995">ERC FRESCO PI</a></div>
+    <div class="components">Ring, Mathematical Components</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/erikmd.png" alt="Érik Martin-Dorel" width="170px"/>
+    </div>
+    <div class="name"><a href="https://www.irit.fr/~Erik.Martin-Dorel/">Érik Martin-Dorel</a></div><div class="nickname">erikmd</div>
+    <div class="position">Associate Professor at IRIT, Toulouse University, France</div>
+    <div class="components">Primitive Types, Docker, Proof-General</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/palmskog.jpg" alt="Karl Palmskog" width="190px"/>
+    </div>
+    <div class="name"><a href="https://setoid.com/">Karl Palmskog</a></div><div class="nickname">palmskog</div>
+    <div class="position">Researcher at the KTH Royal Institute of Technology, Stockholm, Sweden</div>
+    <div class="role">Coq-community manager</div>
+    <div class="components">coq-community, opam, website, Roosterize</div>
+  </div>
+  <div class="person">
+    <div class="picture">
+      <img src="/files/team/cpitclaudel.jpg" alt="Clément Pit-Claudel" width="170px"/>
+    </div>
+    <div class="name"><a href="http://pit-claudel.fr/clement/">Clément Pit-Claudel</a></div><div class="nickname">cpitclaudel</div>
+    <div class="position">PhD Student at MIT, MA, USA</div>
+    <div class="components">Documentation, Alectryon, Proof-General, Company-Coq</div>
```

Comment:
Clément has also contributed to SerAPI, and had a large influence in the early design, in case he'd like it to be added here.